src/esp/bindings/ShortestPathBindings.cpp

```diff
@@ -95,6 +96,12 @@ void initShortestPathBindings(py::module& m) {
       .def("island_radius", &PathFinder::islandRadius, "pt"_a)
       .def_property_readonly("is_loaded", &PathFinder::isLoaded)
       .def_property_readonly("navigable_area", &PathFinder::getNavigableArea)
+      .def_property_readonly(
```

Comment:
These should probably be methods to discourage them from being called in a tight loop

Reply:
Haven't heard that argument before... we could cache it potentially?

Reply:
The general expectation of properties IMO is that they are lightweight,

```python
for vid in vids:
    pathfinder.navmesh_vertices[vid]
```

seems fine while
```python
for vid in vids:
    pathfinder.build_navmesh_vertices()[vid]
```

definitely doesn't.


Yeah, caching would be good.

Reply:
@erikwijmans Done